include/alpaka/dev/DevUniformCudaHipRt.hpp

```diff
@@ -108,18 +116,58 @@ namespace alpaka
 
     namespace trait
     {
+
+        template<typename TApi>
+        static inline void setDeviceProperties(
```

Comment:
This should not be `static`:
```suggestion
        inline void setDeviceProperties(
```